src/include/sys/types.h

```diff
@@ -2,8 +2,32 @@
 #define SYS_TYPES_H
 
 #include <stdint.h>
+#include <stdbool.h>
 
 typedef int64_t time_t;
 typedef int64_t suseconds_t;
 
+#ifndef __is_libc
+typedef struct opt_uint8 {
+    bool is_valid;
```

Comment:
Should it be `has_value` given that it is an optional?

Reply:
Fixed